docs/source/reference/monitoring.md

```diff
@@ -33,9 +33,14 @@ export JUPYTERHUB_METRICS_PREFIX=jupyterhub_prod
 
 would result in the metric `jupyterhub_prod_active_users`, etc.
 
-## Customizing spawn bucket sizes
+## Customizing bucket sizes
 
-As of JupyterHub 5.3, override `JUPYTERHUB_SERVER_SPAWN_DURATION_SECONDS_BUCKETS` env variable in Hub's environment to allow custom bucket sizes. Otherwise default to, [0.5, 1, 2.5, 5, 10, 15, 30, 60, 120, 180, 300, 600, float("inf")]
+As of JupyterHub 5.3, the following environment variables in the Hub's environment can be overridden to support custom bucket sizes:
+
+| Variable                                           | Default                                                                                     |
+| -------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| `JUPYTERHUB_SERVER_SPAWN_DURATION_SECONDS_BUCKETS` | `[0.5, 1, 2.5, 5, 10, 15, 30, 60, 120, 180, 300, 600, float("inf")]`                        |
+| `JUPYTERHUB_SERVER_STOP_DURATION_SECONDS_BUCKETS`  | `[0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10, float("inf")]` |
```

Comment:
This table might give the impression  that the right column is what the environment variable should look like. Maybe use the form of the variable itself:

```
`0.5,1,2.5,5,10,15,30,60,120,180,300,600,inf`
```